Kickstarter-iOS/ViewModels/UpdateViewModel.swift

```diff
@@ -5,15 +5,14 @@ import Prelude
 import ReactiveSwift
 import Result
 
-public struct UpdateData {
-  public let project: Project
-  public let update: Update
-  public let context: Koala.UpdateContext
+fileprivate struct UpdateData {
```

Comment:
you could also kill this `UpdateData` struct entirely and just use a pair for project and update if your heart desires :)